integrations/auth0/client.go

```diff
@@ -79,9 +83,45 @@ func connStatus(i *integration) sdkintegrations.OptFn {
 	})
 }
 
-// TODO(INT-113): Implement
-func connTest(_ *integration) sdkintegrations.OptFn {
+// connTest is an optional connection test provided by the integration
+// to AutoKitteh. It is used to verify that the connection is working
+// as expected. The possible results are "OK" and "error".
+func connTest(i *integration) sdkintegrations.OptFn {
 	return sdkintegrations.WithConnectionTest(func(ctx context.Context, cid sdktypes.ConnectionID) (sdktypes.Status, error) {
-		return sdktypes.NewStatus(sdktypes.StatusCodeUnspecified, "Not implemented"), nil
+		if !cid.IsValid() {
+			return sdktypes.NewStatus(sdktypes.StatusCodeError, "Init required"), nil
+		}
+
+		vs, err := i.vars.Get(ctx, sdktypes.NewVarScopeID(cid))
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		token := vs.Get(authTokenName).Value()
```

Comment:
https://github.com/autokitteh/autokitteh/pull/986#discussion_r1923293521

The whole point of this check is to show that the connection is usable, not that it was created more than an hour ago.